x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -148,6 +148,9 @@
     public static final NamedClusterPrivilege MANAGE_LOGSTASH_PIPELINES = new ActionClusterPrivilege("manage_logstash_pipelines",
         Set.of("cluster:admin/logstash/pipeline/*"));
 
+    public static final NamedClusterPrivilege CANCEL_TASKS = new ActionClusterPrivilege("cancel_tasks",
+        Set.of("cluster:admin/cancel/task"));
```

Comment:
This should be `tasks/cancel` not `cancel/task`.
The unit test fails because it does not use `CancelTasksAction#NAME`.

Reply:
thanks, I pushed https://github.com/elastic/elasticsearch/pull/68679/commits/47cd93cf8501c856fc419ca21665d3238bbb7a2b